### ISSUE TITLE
Improvement of steering management at a standstill

### ISSUE THREAD

**Issue by DragoKeys** [adam10603/AC-Advanced-Gamepad-Assist#35]
Hello, I hope you're well.

Since yesterday, I've been searching for the ideal mod for my PS3 SIXAXIS controller. Two mods caught my attention: yours and "Akey7's Steering-Assist" (A7-Assist - Gamepad FX). I initially tried A7-Assist - Gamepad FX, and I really appreciated how it manages the steering when the car is stationary. Here is a video I made to show you :
https://github.com/adam10603/AC-Advanced-Gamepad-Assist/assets/96720904/13ebaa37-f8f5-480c-a2ba-0056614aec00

When restarting, the wheels return to a straight position (which makes sense), and I find that quite appealing as it adds realism.

However, there are issues when the car starts to skid—it generally recovers on its own. In drifting, once the wheels are fully turned, it becomes impossible to steer further, making it hard to recover the car. At high speeds, there are "shakes" visible in this video (hosted on my drive due to the file size) :
https://kdrive.infomaniak.com/app/share/278166/2b688f0e-d6c0-4b4f-bff5-eea262cf6761

I wonder if, despite the differences in the design of A7-Assist - Gamepad FX, it would be possible to integrate a feature to lock the steering at a standstill and automatically straighten the wheels.

Thank you for reading this, and I look forward to your feedback.

Wishing you a pleasant day ;)
Best regards,

**Comment by adam10603** [comment]
Yeah, A7's script was one of the first ones out there, so it's a lot simpler than mine. 

In my [other similar mod](https://github.com/adam10603/BeamNG-Advanced-Steering) I've made for BeamNG (which is sadly in a half-broken state now after some game updates) I included a "photo mode" which is basically what you're talking about. The idea there was to let you leave the wheels turned for taking pictures and such when the car is stationary. I guess I could consider something similar in the future for AC as well. I'll think about it.
